### PR TITLE
Add new google ads carousel match format.

### PR DESCRIPTION
**Pull request by jameshilliard**

### CODE REVIEW

File: easylist/easylist_general_hide.txt [hunk 1]
```diff
@@ -21271,6 +21271,7 @@
 ##.l-container > #fishtank
 ! Google
 ###atvcap + #tvcap > .mnr-c > .commercial-unit-mobile-top
+###atvcap > div:has(h3[role="heading"] > span:has-text(/^Ads/))
```
Comment (krystian3w) [inline easylist/easylist_general_hide.txt:21274]: - `:has-text(foo)` and `:has(foo)` is no supported by ABP/AdBlock

- omitted domains is no supported by ABP/AdBlock for procedural filters (like `:-abp-has(foo)`)

   - also no supported simplify procedural filters to only TLD like `com##...`

**Review by krystian3w — CHANGES_REQUESTED** [review]

**Comment by krystian3w** [comment]
Filter need move to other file, if no possible remove useage `:-abp-has()` and `:-abp-contains()`

It would also be appropriate to add around ~220 google domains with their various public suffixes ("TLD") before `#?#`.

**Review by krystian3w — CHANGES_REQUESTED** [review]

**Comment by jameshilliard** [comment]
Think it's safe to unconditionally block `#atvcap`?

**Comment by ryanbr** [comment]
We already have `###atvcap + #tvcap > .mnr-c > .commercial-unit-mobile-top` (which #atvcap would overrule.) How safe is blocking #atvcap outright?

**Comment by jameshilliard** [comment]
> How safe is blocking #atvcap outright?

Well the class name sounds to me like it's ad specific and not a random div name, but hard to say for sure.

**Pull request closed without merging**